adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Properties/AssemblyInfo.cs

```diff
@@ -29,16 +29,9 @@
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyProduct("Active Directory Authentication Library")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCulture("")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation. All rights reserved.")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyMetadata("Serviceable", "True")]
+// Version and Metadata are set at build time from msbuild properties defined in the csproj
 
-[assembly: AssemblyFileVersion("3.17.3.0")]
+[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
What is it for?

Reply:
I believe this is related to Windows Security Updates - https://github.com/dotnet/cli/issues/3345

Reply:
Not sure if it applies to our stuff, I believe it is related to GACed assemblies: 


_The CLR identifies Microsoft published NuGet libraries via a special custom attribute created for this purpose. When a library with this attribute is loaded, the CLR records the load event as a marker on the machine, specifically the full assembly name. The marker information never leaves the machine, nor is it sent to Microsoft in any form.

At the point that we release a security update for a NuGet library, Microsoft Update will use the presence of the marker for that library as an indication that a given machine is a candidate for that update. The check for markers is done on the client machine. Markers that don’t have an associated security update are never consulted or used.

Microsoft Update installs updated assemblies into the Global Assembly Cache (GAC), to make them available to all users and apps on the machine that use a given .NET NuGet library. The update includes publisher policy, which is also installed to the GAC. Publisher policy redirects older, vulnerable, versions of libraries to new, updated, versions. The policy is intended to be as targeted as possible, and will not apply to older versions that are not vulnerable, since that is not necessary._